wsd/DocumentBroker.cpp

```diff
@@ -834,6 +840,12 @@ bool DocumentBroker::isAlive() const
     return _childProcess && _childProcess->isAlive();
 }
 
+void DocumentBroker::timeoutNotLoaded(std::chrono::steady_clock::time_point now)
+{
+    if (!_stop && !_poll->isAlive() && !isLoaded() && now - _createTime > std::chrono::seconds(getLimitLoadSecs()))
```

Comment:
Minor: seems we could return a `chrono` type from `getLimitLoadSecs()`, given the conversions done on every usage. Follow-up item.